sidebars.js

```diff
@@ -31,41 +31,86 @@ const sidebars = {
   ourSidebar: [
     'cellular-iot',
     'quickstart',
-    'service-stack-overview',
-    'services',
-    'rest-api',
     {
       type: 'category',
-      label: 'Data Streamer',
+      label: 'Services',
       link: { 
         type: 'generated-index', 
-        title: 'emnify multicloud Data Streamer',
-        slug: 'data-streamer'
+        description: 'emnify offers the following high-level products and services:',
+        slug: 'services'
       },
       items: [
-        'data-streamer/getting-started',
-        'data-streamer/connection-types',
-        'data-streamer/stream-types',
-        'data-streamer/managing-data-streams',
-        'data-streamer/usage',
-        'data-streamer/available-integrations'
-      ],
-    },
-    {
-      type: 'category',
-      label: 'Events',
-      link: {
-        type: 'generated-index', 
-        title: 'emnify system events',
-        slug: '/events'
-      },
-      items: [
-        'events/getting-started',
-        'events/event-types',
-        'events/mno-events',
-        'events/usage'
+        'services/global-iot-sim',
+        'services/global-iot-network',
+        'services/iot-cloud-communication-platform',
+        {
+          type: 'category',
+          label: 'Data Streamer',
+          link: { 
+            type: 'generated-index', 
+            title: 'emnify multicloud Data Streamer',
+            slug: 'data-streamer'
```

Comment:
This slug needs to be updated before merging: 

```suggestion
            slug: 'services/data-streamer'
```